hw/ip/pwm/data/pwm.hjson

```diff
@@ -35,6 +35,7 @@
       desc: "Register write enable for all control registers",
       swaccess: "rw1c",
       hwaccess: "hro",
+      async: "clk_core_i",
```

Comment:
Question for @tjaychen & @Muqing-Liu: Does this `async` tag know that the register also uses `rst_core_ni` for resets?  How does that part work?

Reply:
when you declare a register as async, you must declare it relative to one of the clocks defined [here](https://github.com/lowRISC/opentitan/blob/bb5e76d86570bf6556132dbeb6db64f548f23e8b/hw/ip/pwm/data/pwm.hjson#L5).
This also has associated reset, and that's how the logic picks out which reset to use.